src/oogeso/dto/__init__.py

```diff
@@ -1,2 +1,3 @@
 from .oogeso_input_data_objects import *
+from .oogeso_input_data_objects_multienergy import *
 from .oogeso_output_data_objects import *
```

Comment:
Should import explicitly each class and function. Avoid using from <module> import *

Reply:
Will do this later as part of a general "import *" fix